flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/over/AbstractRowTimeUnboundedPrecedingOver.java

```diff
@@ -68,6 +70,17 @@
 
 	protected transient AggsHandleFunction function;
 
+	// ------------------------------------------------------------------------
+	// Metrics
+	// ------------------------------------------------------------------------
+	protected static final String LATE_ELEMENTS_DROPPED_METRIC_NAME = "numLateRecordsDropped";
+	private transient Counter numLateRecordsDropped;
+
+	@VisibleForTesting
+	public Counter getCounter() {
```

Comment:
could be `protected`?

Reply:
Reason for consideration of future sub-class extension?In that way,i'll change the constant to `private` in `RowTimeRangeBoundedPrecedingFunction` and  `RowTimeRowsBoundedPrecedingFunction`.
WDYT

Reply:
we could not access it if it's `private`, `protected` methods can be accessed in tests.

Reply:
we can access it by `getCounter()`

Reply:
I mean `protected getCounter()`

Reply:
got it